docs-src/modules/ROOT/pages/introduction.adoc

```diff
@@ -114,12 +114,12 @@ aliases used assumes you have the following namespace requires:
     [com.wsscode.common.async-cljs :refer [let-chan <!p go-catch <? <?maybe]]))
 ```
 
-So, any time you see the usage of a namespace in a keyword or function like `p/parser` or `::p/reader` you should remember
+So, any time you see the usage of a namespace in a keyword or a function like `p/parser` or `::p/reader` you should remember
 that these are the namespaces involved.
 
 == Presentations
 
-If you like to learn by seeing presentations, there are some that mention it:
+If you like to learn by watching videos, there are two presentations on Pathom:
```

Comment:
the list has three items, I would rather keep `some` so the number doens't need to be exact

Reply:
Ah, yes. Might I suggest you use, `some` for this :)